manifests/l/Lexicool/LexibarES/1.4f/Lexicool.LexibarES.locale.fr-FR.yaml

```
@@ -0,0 +1,25 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Lexicool.LexibarES
+PackageVersion: 1.4f
+PackageLocale: fr-FR
+Publisher: Lexicool
+PublisherUrl: https://www.lexicool.com/
+PublisherSupportUrl: https://www.lexicool.com/about.asp
+PrivacyUrl: https://www.lexicool.com/about.asp
+Author: Lexicool
+PackageName: Lexibar Spanish
+PackageUrl: https://www.lexicool.com/lexibar_spanish_special_keyboard_characters.asp
+License: Copyright © lexicool 2000-2022
+LicenseUrl: https://www.lexicool.com/about.asp
+Copyright: Copyright © lexicool 2000-2022
+CopyrightUrl: https://www.lexicool.com/about.asp
+ShortDescription: Lexibar pour caractère spéciaux clavier Espagnol
+Description: Lexibar Espagnol permet l'accès rapide aux caractères spéciaux de l'espagnol qui ne sont pas présents sur le clavier
+Moniker: LexibarES

```

```suggestion
Moniker: lexibar-es
```
A moniker is a special type of tag. It should be written in lowercase with hyphens instead of space.